requirements/base.txt

```diff
@@ -1,6 +1,6 @@
 celery
 django>=1.7,<1.9
-django-guardian>=1.3
+django-guardian>=1.3,<=1.4.1
```

Comment:
Ok well what if we take out this change and get https://github.com/nyaruka/smartmin/pull/71 merged ?